src/Services/ModerationLogService.cs

```diff
@@ -64,17 +64,23 @@ private Embed CreateEmbedResponse(ModerationLogEntry logEntry)
             EmbedBuilder builder = new EmbedBuilder();
 
             StringBuilder author = new StringBuilder($"[{logEntry.ActionType.Humanize()}]");
+            List<string> footnotes = new List<string>();
             
             if (logEntry.HasTarget)
             {
                 if (logEntry.Target != null)
-                    author.Append($" {logEntry.Target.ToString()}");
+                    author.Append($" {logEntry.Target.Username}#{logEntry.Target.Discriminator}");
                 else
-                    author.Append($" {logEntry.TargetMention}");
+                    author.Append($" {logEntry.TargetID}");
+                
+                footnotes.Add($"User ID: {logEntry.TargetID}");
```

Comment:
I don't think adding "User ID" is necessary, it's a well known value format. And it makes the footer cleaner.

Reply:
Changed in a94d6c70e14a004bfff29c930a87852fcfc1ec00